### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by praveenyr**
**Description:** Completed Sign up and Sign in features.

### CODE REVIEW

File: app/assets/javascripts/welcome.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/welcome.coffee]: @praveenyr please delete this, we're not using it.

File: app/helpers/welcome_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (brittanyjspringer) [inline app/helpers/welcome_helper.rb]: @praveenyr please delete this, we're not using it.

File: test/fixtures/users.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (brittanyjspringer) [inline test/fixtures/users.yml]: @praveenyr please delete this, we're not using it.

File: test/controllers/welcome_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```
Comment (brittanyjspringer) [inline test/controllers/welcome_controller_test.rb]: @praveenyr please delete this, we're not using it.

File: test/models/user_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (brittanyjspringer) [inline test/models/user_test.rb]: @praveenyr please delete this, we're not using it.

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]
Praveen, I've requested several files to be deleted. Also, please add WelcomeController rspec tests.

**Comment by praveenyr** [comment]
@wonderwoman13 all set :)

Happy 4th to you too.

**Comment by brittanyjspringer** [comment]
Excellent, you too!

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**